product_docs/docs/pgd/5/proxy/index.mdx

```diff
@@ -0,0 +1,119 @@
+
+## Configuration
+
+Proxy connects to BDR database for its internal operations, like getting proxy configs, getting write leader details, etc. Therefore, it needs a list of endpoints/dsn to connect to BDR nodes. Proxy expects these configurations in a local config file `pgd-proxy-config.yml`. Following is a functional example of `pgd-proxy-config.yml` file:
+
+```yaml
+cluster:
+  name: my_pgd_cluster
+  endpoints:
+    - "host=bdr-a1 port=5432 dbname=bdrdb user=pgdproxy "
+    - "host=bdr-a3 port=5432 dbname=bdrdb user=pgdproxy "
+    - "host=bdr-a2 port=5432 dbname=bdrdb user=pgdproxy "
+  proxy:
+    name: "proxy-a1"
+```
+
+The `cluster.endpoints` and `cluster.proxy.name` are mandatory fields in the config file. Proxy always tries to connect to the first endpoint in the list, if it fails it tries the next endpoint and so on.
+
+PGD Proxy searches for `pgd-proxy-config.yml` in the following locations (precedence order - higher to lower):
+
+   1. "-f config-file" (e.g. 'pgd-proxy -f /opt/my-config.yml')
+   2. "/etc/edb/pgd-proxy" (default)
+   3. "$HOME/.edb/pgd-proxy"
+
+The `pgd-proxy-config.yml`, is located in the `/etc/edb/pgd-proxy` directory, by default. If you rename the file or move it to another location, specify the new name and location using the `-f` flag in the `pgd-proxy.service` file.
+
+### Proxy user authentication
+
+The password for the database user which is given in the endpoint should be configured properly on the proxy machine using [The Password File](https://www.postgresql.org/docs/current/libpq-pgpass.html) file. If the PGD Cluster is created by TPAexec, the default configuration file and the password file file is generated automatically, but with a manual installation to a EDB Postgres Distributed cluster you need to create the config file and setup authentication manually.
+
+
+## PGD Proxy service file
+
+PGD Proxy is preferably run as a systemd service. Following is the sample service file for EDB Postgres Extended Server and Postgres Community.
+
+*Note:* For EDB Postgres Advanced Server please change User and Group from `postgres` to `enterprisedb`.
+
+```
+[Unit]
+Description=PGD Proxy
+
+[Service]
+Type=simple
+User=postgres
+Group=postgres
+Restart=on-failure
+RestartSec=1s
+ExecStart=/usr/bin/pgd-proxy -f /etc/edb/pgd-proxy/pgd-proxy-config.yml
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=pgd-proxy
+
+[Install]
+WantedBy=multi-user.target
+```
+
+
+## Managing PGD Proxy
+
+PGD CLI provides few commands viz., `create-proxy`, `show-proxy`, `set-proxy-options` and `delete-proxy` to manage Proxies in PGD cluster. See [PGD CLI](/pgd/latest/cli/command_ref) for more information.
+
+See [Connection management](/pgd/latest/routing) for more information on BDR side of configuration and management of PGD Proxy.
+
+
+## Proxy log location
+### syslog
+- Debian based - /var/log/syslog
+- Red Hat based - /var/log/messages
+
+Use `journalctl` command to filter and view logs for troubleshooting Proxy.
```

Comment:
I think we can add those complete example which has unit file name etc..

```
Use following commands -
journalctl -u pgd-proxy -n100 -f
journalctl -u pgd-proxy --since today
journalctl -u pgd-proxy --since "10 min ago"
journalctl -u pgd-proxy --since "2022-10-20 16:21:50" --until "2022-10-20 16:21:55"


```